centaur/src/main/resources/standardTestCases/read_write_map/read_write_map.wdl

```diff
@@ -26,10 +26,30 @@ task read_map {
   }
 }
 
+task assert_write_map_terminal_newline {
```

Comment:
This feels like it would be better, and easier to add more edge cases to, as a unit test of the `def tsvSerialize: Try[String]` than here an integration test

Reply:
+1